docs/howtos/locally-published-components-in-fenix.md

```diff
@@ -8,44 +8,63 @@ from there.
+
+## Setup 2 x `local.properties`
+
+Edit (or create) the file `local.properties` in each of the repos:
+
+### app-services
+
+In the root of the app-services repo:
+
+#### rust.targets
 
 Both the auto-publishing and manual workflows can be sped up significantly by
 using the `rust.targets` property which limits which architectures the Rust
-code gets build against.  You can set this property by creating/editing the
-`local.properties` file in the repository root and adding a line like
+code gets build against.  Adding a line like
 `rust.targets=x86,linux-x86-64`.  The trick is knowing which targets to put in
 that comma separated list:
 
-  - Use `x86` for running the app on most emulators (in rare cases, when you have a 64-bit emulator, you'll want `x86_64`)
+  - Use `x86` for running the app on most emulators on Intel hardware (in rare cases, when you have a 64-bit emulator, you'll want `x86_64`).
+  - Use `arm64` for emulators running on Apple Silicon Macs.
   - If you're running the `android-components` or `fenix` unit tests, then you'll need the architecture of your machine:
     - OSX running Intel chips: `darwin-x86-64`
     - OSX running M1 chips: `darwin-aarch64`
     - Linux: `linux-x86-64`
 
-## Using the auto-publishing workflow
+eg, on a Mac your `local.properties` file will have a single line, `rust.targets=darwin-aarch64,arm64`
 
-mozilla-central has support for automatically publishing and including a local development version of application-services in the build.
-This is supported for most of the Android targets available in mozilla-central including Fenix - this
-doc will focus on Fenix, but the same general process is used for all. The workflow is:
+### mozilla-central
 
-1. Ensure you have a regular build of Fenix working from mozilla-central and that you've done a `./mach build`
-1. Ensure you have a regular [build of application-services working](../building.md).
-1. Edit (or create) the file `local.properties` - this can be in the root of the mozilla-central checkout,
-   or in the project specific directory (eg, `mobile/android/fenix`) and tell it where to
-   find your local checkout of application-services, by adding a line like:
+`local.properties` can be in the root of the mozilla-central checkout,
+or in the project specific directory (eg, `mobile/android/fenix`) and you tell it where to
+find your local checkout of application-services by adding a line like:
+
+`autoPublish.application-services.dir=path/to/your/checkout/of/application-services`
+
+Note that the path can be absolute or relative from `local.properties`. For example, if `application-services`
+and `mozilla-central` are at the same level, and you are using a `local.properties` in the root of mozilla-central,
+the relative path would be `../application-services`
 
-   `autoPublish.application-services.dir=path/to/your/checkout/of/application-services`
+## Build and test your Fenix again.
```

Comment:
Not sure, but should we add that:

If you have trouble building `m-c` inside Android Studio, try:

1. Have a virtual device running inside Android Studio
2. Do `./mach build` 
3. And then `./mach run --app=fenix`, which installs and runs Fenix inside the running emulator.

Reply:
huh - I didn't know that worked :) I thought that `run` command always started a stand-alone emulator. Ideally though we'd find canonical docs for building Fenix and link to that, but adding this here for now sgtm